test/unit/checkpoint/graphql/__snapshots__/utils.test.ts.snap

```diff
@@ -0,0 +1,51 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`toGql should work 1`] = `
+"type Query {
+  votes(
+    first: Int
+    skip: Int
+    orderBy: String
+    orderDirection: String
+    where: WhereVote
+  ): [Vote]
+  vote(id: String): Vote
+}
+
+
+input WhereVote {
+  id: null
+  id_in: [null]
```

Comment:
this looks off, the types generated are `null`, I'm assuming they should be `Int` based on the `type Vote` defined. @bonustrack is this the expected behavior or it is a bug we should look into?

Reply:
That's right, based on the schema defined it should return a type `Int`. I think the setup I've done assume the id is a String see https://github.com/snapshot-labs/sx-api/blob/master/src/checkpoint/graphql/utils.ts#L86 but there is no reason to enforce that, a Int should work as id too.